pkgs/development/python-modules/pandoc/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, plumbum
+, ply
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "boisgera";
+    repo = pname;
+    rev = version;
```

This helps to avoid branch/tag confusion. It will make bulk updates smoother.

```suggestion
    rev = "refs/tags/${version}";
```

---

Changed.